### DISCUSSION TITLE
请问可以添加后台运行相关功能吗？

### DISCUSSION THREAD

**Discussion by h0l1day117** [Meloong-Git/PCL#2481] [提问] [answered]
就是点右上角叉号之后会提示直接关闭还是后台运行，在电脑右下角“隐藏的图标”那个箭头可以再打开。就像其他很多软件一样。也可以设置不再询问

**Comment by zkitefly** [comment]
新功能就提个 issue 吧

还有我觉得龙猫应该不会加的

**Reply by h0l1day117** [reply]
还是挺方便的功能，而且挺常见的。主要是任务栏图标太多看着难受，一般暂时没用的窗口我都会关掉

**Comment by MoYuan-CN** [comment, marked as answer]
参考下 **[拒绝]** #1161（#772 #1133）的启动器支持缩小到托盘？

**Reply by h0l1day117** [reply]
喔，我居然没看到，谢谢
